optaplanner-core/src/main/java/org/optaplanner/core/impl/partitionedsearch/scope/PartitionChangeMove.java

```diff
@@ -85,13 +86,13 @@ protected void doMoveOnGenuineVariables(ScoreDirector<Solution_> scoreDirector)
     }
 
     @Override
-    public boolean isMoveDoable(ScoreDirector<Solution_>  scoreDirector) {
+    public boolean isMoveDoable(ScoreDirector<Solution_> scoreDirector) {
         return true;
     }
 
     @Override
-    public PartitionChangeMove<Solution_> createUndoMove(ScoreDirector<Solution_> scoreDirector) {
-        throw new UnsupportedOperationException();
+    public NoChangeMove<Solution_> createUndoMove(ScoreDirector<Solution_> scoreDirector) {
+        return new NoChangeMove<>();
```

Comment:
Lets add a comment above: // TODO not yet implemented, returns null to fail fast if it is used